docs/language/crypto.mdx

```diff
@@ -142,8 +141,7 @@ publicKey.publicKey = []                                            // Not allow
 publicKey.publicKey[2] = 4      // No effect
 ```
 
-The validity of a public key can be checked using the `isValid` field.
-Verifications performed with an invalid public key (using `verify()` method) will always fail.
+Invalid public keys cannot be constructed so public keys are always valid.
```

Comment:
Docs change.

Are any other docs changes needed? I see CHANGELOG has some relevant info but I'm not sure when that gets updated.

Reply:
No, this should be all. It gets pulled by github.com/onflow/flow and ends up on docs.onflow.org